staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -966,6 +971,14 @@ type MemorySwapConfiguration struct {
 	SwapBehavior string `json:"swapBehavior,omitempty"`
 }
 
+type CrashLoopBackOffConfig struct {
+	// maxSeconds is the maximum number of seconds the backoff delay can accrue
+	// to for container restarts
```

Comment:
end in `.`

document default, min, max

Reply:
Fixed in 6692dda

Reply:
TBD default